tree/treeplayer/inc/ROOT/TCsvDS.hxx

```diff
@@ -33,13 +33,15 @@ private:
    static const std::map<ColType_t, std::string> fgColTypeMap;
 
    unsigned int fNSlots = 0U;
-   std::string fFileName;
-   char fDelimiter;
+   std::unique_ptr<std::ifstream> fStream;
+   const char fDelimiter;
+   const Long64_t fLinesChunkSize;
+   ULong64_t fEntryRangesRequested = 0ULL;
```

Comment:
this is an `unsigned long` but if I'm not mistaken it is only ever used to check whether it is `0` or `1`, so maybe it would be clearer to make it a bool (`fFirstEntryRangeRequested`)?

Reply:
After the fix I did when calculating the offset of the entry in the current chunk, it becomes necessary that `fEntryRangesRequested` stays as an `unsigned long` (it is actually used to calculate the offset)